src/test/scala/com/microsoft/hyperspace/index/rules/JoinIndexRuleTest.scala

```diff
@@ -401,7 +401,12 @@ class JoinIndexRuleTest extends HyperspaceRuleTestSuite with SQLHelper {
       updatedPlan: LogicalPlan,
       indexPaths: Seq[Path]): Unit = {
     assert(treeStructureEquality(originalPlan, updatedPlan))
-    assert(basePaths(updatedPlan) == indexPaths)
+    val rootPaths = getRootPaths(updatedPlan)
+    assert(rootPaths.length == indexPaths.length)
+    assert(rootPaths.zip(indexPaths).forall {
+      case (a, b) =>
+        a.toString.startsWith(b.toString)
```

Comment:
Looks like `indexPaths` can be updated to have full path to the file?

Reply:
thanks @imback82 , reverted the tests to use full paths